bin/ubuntu-core-initramfs

```diff
@@ -326,6 +326,7 @@ def install_systemd_files(dest_dir, sysroot):
     # * Configuration for systemd-tmpfiles
     # TODO: some of this can be cleaned up
     to_include = re.compile(r"^(/usr)?/lib/systemd/system|"
+                            r"/usr/share/dbus-1/.*/org.freedesktop.systemd1\..*|"
```

Comment:
It seems we are copying `services/org.freedesktop.systemd1.service`, but this is for user session and is useless in initramfs. It is `system-services/org.freedesktop.systemd1.service` that is used. Otherwise it looks fine.

Reply:
Good point, I've stopped copying that one